$OpenBSD: patch-lib_rubygems_rb,v 1.1 2014/01/11 22:29:31 jeremy Exp $

Backport rubygems commit b1bf2e63b69 to make gem extensions directory
not include ruby tiny version.

--- lib/rubygems.rb.orig	Thu Jan  9 19:53:35 2014
+++ lib/rubygems.rb	Thu Jan  9 19:53:44 2014
@@ -785,8 +785,7 @@ module Gem
   # Returns a String containing the API compatibility version of Ruby
 
   def self.ruby_api_version
-    @ruby_api_version ||=
-      "#{ConfigMap[:MAJOR]}.#{ConfigMap[:MINOR]}.#{ConfigMap[:TEENY]}"
+    @ruby_api_version ||= RbConfig::CONFIG['ruby_version'].dup
   end
 
   ##
